### DISCUSSION TITLE
Apache Skywalking - Zipkin Receiver Question

### DISCUSSION THREAD

**Discussion by lstephe** [apache/skywalking#9101] [Q&A] [answered]
We are trying to setup the zipkin receiver;  it is not working;  I want to be able to get the zipkin receiver to work to push traces to Skywalking for one application;  for this one application traces are not visible in Skywalking natively.  

I saw this in the documentation link here: https://skywalking.apache.org/docs/main/latest/en/setup/backend/zipkin-trace/ ‘Once this receiver and storage are activated, SkyWalking’s native traces would be ignored, and SkyWalking wouldn’t analyze topology, metrics, and endpoint dependency from Zipkin’s trace.’  

Does the above stmt only apply to the one application where traces are being pushed to Skywalking via the zipkin receiver?  Or will it apply to all applications currently hosted in Skywalking?  Does it make sense what I can asking?  

Thanks

**Comment by wu-sheng** [comment, marked as answer]
It is not working, we are implementing the Zipkin alternative server in a totally different way. See https://github.com/apache/skywalking/issues/8932

**Reply by wu-sheng** [reply]
We could support both after we finish this, but we wouldn't analyze metrics from Zipkin spans, as they are highly likely sampled in a low rate, and topology analysis is not product env friendly as it is a pure tracing format, not an APM level tracing. Read this paper, https://skywalking.apache.org/docs/main/latest/en/papers/stam/
